docs/chapters/sdk/using-the-sdk-to-crosscompile.rst

```diff
@@ -0,0 +1,21 @@
+Using the SDK to cross compile
+==============================
+
+Cross compiling with help of the SDK is not much different to normal compiling, there are more pitfalls but let's get on with the happy cases for now.
+
+Sourcing Environment
+--------------------
+
+Once you have installed the SDK you need to source it, which means that the all the right environment variables will be set in your terminal session so the right compiler and sysroot will be used during the compilation.
+
+.. code-block:: bash
+                
+    cd ~/sdk
+    source environment-setup-corei7-64-poky-linux
+
+This sourcing of the environment is only active in this one terminal session, if you open a new one you have to source it again. To check if you are in a sourced environment you can print out the environment with `env`, if you see environment variables like `OECORE_SDK_VERSION` then you are in a sourced environment.
```

Comment:
Maybe it would be worth writing the command as a one liner, e.g.:  `env | grep OECORE`?